vendor/github.com/minio/minio-go/api.go

```diff
@@ -99,7 +99,7 @@ type Options struct {
 // Global constants.
 const (
 	libraryName    = "minio-go"
-	libraryVersion = "4.0.7"
+	libraryVersion = "4.0.9"
```

Comment:
This line shows that we skipped `4.0.8` version.
Either PR title, which says `vendorize minio-go to version 4.0.8`, is wrong, or `libraryVersion` in minio-go code is set to `4.0.9` version by mistake.